Gruntfile.js

```diff
@@ -70,7 +70,8 @@ const configureGrunt = function (grunt) {
                     'content/themes/casper/assets/js/*.js'
                 ],
                 options: {
-                    livereload: true
+                    livereload: true,
+                    interval: 5007
```

Comment:
Out of interest, is there something special about `5007`?

Reply:
@allouis it's the example value given in the docs https://github.com/gruntjs/grunt-contrib-watch#why-is-the-watch-devouring-all-my-memorycpu

@gargol I would suggest something smaller than 5secs? Seems like it could make development a bit annoying

Reply:
I couldn't find any reference in the 'old node pooling default'  for this 5007 magic number. Will dig around, but agree on 5s being a bit too long. Maybe 500ms would be better? Just enough to not notice it?

Reply:
500ms sounds more sane as long as we still see the CPU usage benefits

Reply:
Experimented with a 500ms interval, the CPU usage for my machine was under 1%. Think it's good enough result. The delay is barely affecting the dev flow because restart for express server or rebuild of the client still takes a couple of seconds.